tests/notifier/notifiers/cscc_notifier_test.py

```diff
@@ -239,3 +240,66 @@ def test_beta_api_is_invoked_correctly(self):
         _, kwargs = call
         
         self.assertEquals('111', kwargs['source_id'])
+
+    def test_outdated_findings_are_found(self):
+
+        NEW_FINDINGS = [['f66618c04c057e3af5f6fae01b2d1abc',
+                            {'category': 'BUCKET_VIOLATION',
+                             'resource_name': 'organization/660570133860/project/inventoryscanner-henry/bucket/isthispublic/',
```

Comment:
Please do not use our real org data, like the org id here.  Scrub them all.

Reply:
Done